services/web-app/pages/assets/index/index.module.scss

```diff
@@ -51,19 +51,77 @@
  */
 
 .subheading {
-  padding: spacing.$spacing-10 0;
+  padding: (spacing.$spacing-08 + spacing.$spacing-06) 0 spacing.$spacing-08;
```

Comment:
is there a reson we use (`$spacing-08` (40px) + `$spacing-06` (24px)) instead of using $spacing-10 (64px)?

Reply:
Just updated in my PR to this branch. pairing wiht @andreancardona now 😄

Reply:
Just updated in my PR to this branch. pairing wiht @andreancardona now 😄